packages/documents/src/config/routes.ts

```diff
@@ -64,6 +65,15 @@ export const getRoutes = () => {
         tags: ['api']
       }
     },
+    // delete a document
+    {
+      method: 'DELETE',
+      path: `/${MINIO_BUCKET}/{fileUri}`,
```

Comment:
```suggestion
      path: `/files/{filename}`,
```

Could we do this instead?  I added a similar endpoint in the file upload PR 

```
    {
      method: 'GET',
      path: '/files/{filename}',
      handler: fileExistsHandler,
      config: {
        tags: ['api']
      }
    },
```